terraform/bootstrap/create_service_account.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+org="gsa-tts-devtools-prototyping"
```

Comment:
suggestion (non-blocking): It might be good to remove the DevTools specific defaults, like the org name.